chain-tx-enclave-next/tdbe/enclave-app/src/sgx_module.rs

```diff
@@ -0,0 +1,262 @@
+mod handler;
+
+pub use rs_libc::alloc::*;
```

Comment:
I haven't tried, but wonder if this is still necessary? With the upstream(-ish) rust-secp256k1, allocations+abort... are done in Rust, so no need for special patches and libc

Reply:
Does ring need libc allocs?

Reply:
I'm investigating this. Will create a new PR if it isn't necessary anymore.